index.html

```diff
@@ -389,6 +389,12 @@ <h2 class="title-bar-item">JSON Path Finder</h2>
   }
 });
 
+document.querySelectorAll(".big-box").forEach((el) => { el.style.height= (window.innerHeight - 90) + "px" });
```

Comment:
small request: since this code is duplicated on line 395, could you wrap it in its own function? if you name it `sizeBigBoxesToFitScreen` (or something along those lines) it'll have the added benefit of being self-documenting.

Reply:
formatting nit:
```suggestion
document.querySelectorAll(".big-box").forEach((el) => { el.style.height = (window.innerHeight - 90) + "px" });
```